server/auth/gitlab/gitlab.go

```diff
@@ -1,13 +1,18 @@
 package gitlab
```

Comment:
This file looks unrelated to the change? Oh I see, this includes the gitlab changes too. Can you yank those from this PR easily? If not a rebase on this branch after merging the other PR might be the way to go

Reply:
I needed it in order to test as the current gitlab support is broken. I can break it out, yes.

Reply:
Actually could you just merge the gitlab PR in and then I'll merge those changes in to this?

Reply:
Go for it